common/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -261,8 +261,16 @@ message LoadFileRequest {
 
 // A subtask of a load file request. either a load data or load metadata.
 message LoadSubTask {
-  optional Block block = 1;
-  optional UfsStatus ufs_status = 2;
+  optional LoadDataSubTask load_data_subtask = 1;
+  optional UfsStatus load_metadata_subtask = 2;
```

Comment:
in that sense its better to make a LoadMetadataSubTask in future refactor? but no need to address in current PR

Reply:
done, avoid future incompatibility